cmd/schematic/schematic.go

```diff
@@ -70,6 +70,7 @@ func main() {
 
 	code, err := s.Generate()
 	if err != nil {
+		fmt.Fprintf(os.Stderr, "%s\n", code)
```

Comment:
If there was a compile error in the generated code, the code itself would not be logged -- only the error. This change is unrelated generally to the changes here but was _required_ for actually debugging the issue. Without it, the only output would be:

```
schematic: 3300:25: illegal character U+003F '?' (and 6 more errors)
```